scripts/extractEdition.py

```diff
@@ -0,0 +1,46 @@
+from elasticsearch_dsl import Q
+from managers import DBManager
+
+from model import Edition
+
+#Extracting the edition number from the edition_statement column of the DRB database to fill out the edition column
+def main():
+
+    dbManager = DBManager(
+        user='',
+        pswd='',
+        host='',
+        port='',
+        db=''
```

Comment:
just an improvement suggestion for making scripts like this easier to run: set it up so that these are loaded from the CLI, or borrow the environment loading script from `main.py` and use environment variables

(that way you don't have to remove the secrets before committing)